aspnetcore/security/cors/sample/Cors/WebAPI/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace WebAPI
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateWebHostBuilder(args).Build().Run();
+        }
+
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+                .UseStartup<StartupTest>();
+                //.UseStartup<StartupTestMultiPolicy>();
```

Comment:
Remove?

Reply:
I have multiple StartUp classes and I uncomment the one I want to test.